File: SKIBin_old/app/src/main/java/com/h2Invent/skibin/FragmentSettings.java
package com.h2Invent.skibin;

import android.content.Context;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

public class FragmentSettings extends Fragment {

    public static final String TITLE = "Einstellungen";


    public static FragmentSettings newInstance() {

        return new FragmentSettings();
    }

    private OnItemSelectedListener listener;

    public interface OnItemSelectedListener {


    }
    // Store the listener (activity) that will have events fired once the fragment is attached

    @Override
    public void onAttach(@NonNull Context context) {
        super.onAttach(context);

        if (context instanceof OnItemSelectedListener) {
            listener = (OnItemSelectedListener) context;
        } else {
            throw new ClassCastException(context.toString()
                    + " must implement MyListFragment.OnItemSelectedListener");
        }

    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_settings, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);


    }

}

File: app/src/main/java/com/h2Invent/skibin/FragmentSettings.kt
package com.h2Invent.skibin

import android.content.Context
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment

class FragmentSettings : Fragment() {
    private var listener: OnItemSelectedListener? = null

    override fun onAttach(context: Context) {
        super.onAttach(context)
        listener = context as? OnItemSelectedListener
            ?: error("Parent activity must implement OnItemSelectedListener")
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
        return inflater.inflate(R.layout.fragment_settings, container, false)
    }

    interface OnItemSelectedListener

    companion object {
        const val TITLE = "Einstellungen"
        fun newInstance() = FragmentSettings()
    }
}
